tests/integration_tests/test_tensorboard.py

```diff
@@ -10,13 +10,14 @@
 
 
 def _objective_func(trial: optuna.trial.Trial) -> float:
+    v = trial.suggest_int("v", 1, 10, log=True)
```

Comment:
How about adding the following to verify both `log=False` and `log=True`?
```suggestion
    u = trial.suggest_int("u", 0, 10, step=2)
    v = trial.suggest_int("v", 1, 10, log=True)
```